Scripts/Packaging/build_packages.sh

```diff
@@ -23,19 +23,101 @@
 
 set -e
 
+# Default values
 GIT_TOP_LEVEL=$(git rev-parse --show-toplevel)
+ROCM_EXAMPLES_ROOT="$GIT_TOP_LEVEL"
```

Comment:
```suggestion
SRC_DIR="$GIT_TOP_LEVEL"
```
Changing to SRC_DIR to match other path related options. Maybe we should move it lower to also group them all together. NOTE: I haven't changed subsequent references.